spark/check.py

```diff
@@ -367,7 +367,16 @@ def _mesos_init(self, master_address):
                 app_name = app_json.get('name')
 
                 if app_id and tracking_url and app_name:
-                    running_apps[app_id] = (app_name, tracking_url)
+                    spark_ports = instance.get('spark_ui_ports')
+                    if spark_ports is None:
```

Comment:
Do we want to accept every ports if the `spark_ports` is an empty list ?

Reply:
I think if the intended use case is to state which ports to include & you make `spark_ui_ports` an empty list, it would make sense to expect that _no_ ports are included... vs if you don't have `spark_ui_ports` set at all, then there's no filter so we accept every port